internal/eval/eval.go

```diff
@@ -588,6 +588,142 @@ func evalJSONARRLEN(args []string, store *dstore.Store) []byte {
 	return clientio.Encode(arrlenList, false)
 }
 
+func evalJSONARRPOP(args []string, store *dstore.Store) []byte {
+	if len(args) < 1 {
+		return diceerrors.NewErrArity("json.arrpop")
+	}
+	key := args[0]
+
+	var path string = defaultRootPath
+	if len(args) >= 2 {
+		path = args[1]
+	}
```

Comment:
this pr looks good. I have a few comments.  
in redis. if the key neither is an array nor exist . should raise an error.
`

    127.0.0.1:6379> JSON.SET user $ '{"a":[{"b":2},{"g":[1,2]},3]}'
    OK
    127.0.0.1:6379> JSON.ARRPOP user1
    (error) ERR Path '$' does not exist or not an array
    127.0.0.1:6379> JSON.ARRPOP user
    (error) ERR Path '$' does not exist or not an array
    127.0.0.1:6379> JSON.SET user $ '[{"b":2},{"g":[1,2]},3]'
    OK
    127.0.0.1:6379> JSON.ARRPOP user
    "3"
  

`

Reply:
Thanks @jujiale. Fixed this.